Derpitcraft/README.md

```diff
@@ -1,105 +1,65 @@
-
-10. **IGN Policy**: We **DON'T** support changing your IGN mid-season. We are a **cracked server**, and you will **lose your items and player details** if you change your IGN. Do not change your IGN once you start playing with a specific IGN. *Raise a ticket if this is absolutely necessary.*
-
+    **Proximity Chat System**: Derpitcraft has a proximity chat system powered by the *Simple Voice Chat* mod. Download this mod from https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat to play with Proximity Chat on Derpitcraft. It is optional. To read more on how to use this system, read the guide on it below.
+10. **IGN Policy**: We **DON'T** support changing your IGN mid-season. We are a **cracked server**, and you will **lose your items and player details** if you change your IGN. Do not change your IGN once you start playing with a specific IGN. **Raise a ticket if this is absolutely necessary.**
 11.  **Lag Policy**: If you are experiencing lag, please wait a while for it to pass. If it stays and worsens, ping a moderator after **3 minutes** of continuous lag. Keep the following in your mind.
-
-     - If you think you have died due to a lag spike/ TPS drop, we will require **videographic evidence** to confirm it and restore your items. Without video, we cannot restore your items.  
+     - If you think you have died due to a lag spike, TPS drop(under 5 TPS), or bad ping(more than 1000 ms), we will require **videographic evidence** to confirm it and restore your items. Without video, we cannot restore your items.  
   
      - Remember, lag is both **server-side** and **client-side**. We can only compensate for **server-side** lag, and that also, to an extent. It is recommended to keep your X, Y, and Z coordinates on the screen at all times by either using a client-side Mod, or using the *coordinates HUD datapack* installed on the server (Read the how to guide on **commands** for more information).
   
      - This is a **server**, and all items will **despawn** after **5 minutes**, regardless of if you are online or not. The items may even despawn in chunks that you think are unloaded, but get loaded by somebody else. **It is recommended to assume that all items despawn after 5 minutes**. We cannot provide you your items back if they have despawned.
-
 This list is ***not comprehensive***. If you think something may not be allowed, ask us by raising a ticket. If you have any other questions, please raise a ticket. The use of loopholes in these rules is not permitted, and will result in punishment. The say of the moderators is final in all cases. 
-
 ## Compatibility Issues on Bedrock
-
 There might be a few compatibility issues with the server, since we support both Java and Bedrock. None of these issues pre gameplay. You may need to ask a Java edition player to *create maps*, etc. If you are experiencing issues with the server, please mention a moderator.
```

Comment:
breaks shud be break

Reply:
```suggestion
There might be a few compatibility issues with the server, since we support both Java and Bedrock. None of these issues break gameplay. You may need to ask a Java edition player to *create maps*, etc. If you are experiencing issues with the server, please mention a moderator.
```